### PR TITLE
Remove auth, add forms endpoints, and update the readme file

### PR DESCRIPTION
**Pull request by nimlevy**
In this PR we:
1. Remove the authentication mechanism and no longer require a secret
2. Add examples to the three new forms endpoints
3. Update the documentation in the README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,279 +1,105 @@
...
-__Step 1 is Requesting a Bid__ - you submit a Zip Code and Service Category, API returns if there is an available lead Buyer.
-* `/examples/request-bid.php`
-  * Use the zip code `11111` to create a test request.
-  * Or use an actual zip code `90210` to see a real response
-
-__Step 2 is Accepting the Bid__ - you submit a Request ID, the API returns a Tracking Phone Number.
-* `/examples/request-accept.php`
-  * Use the request id `0` to accept a test request and receive a phone number for testing routing numbers.
-  * Calling the test number will direct you to an automated voice mail indicating success.
+To understand our API, see our documentations:
```
Comment (john-servicedirect) [inline README.md]: See full documentation for our APIs here:

File: README.md [hunk 2]
```diff
@@ -1,279 +1,105 @@
...
+  * `POST /partners/forms/{lead_token}/sold_price` - report a lost bid winning price
+
+As well as the following dynamically changing open endpoints to get Service Direct's latest availabilities:
+
+* Service Category mapping; the response includes the service_category_id that 
+  * Phones: https://api.servicedirect.com/resources/service_categories?is_marketplace=1
+  * Forms: https://api.servicedirect.com/resources/service_categories?is_marketplace=1&is_mp_forms=1
+* Finding the highest Cost Per Lead per Zip Code:
+  * `https://api.servicedirect.com/cpl/industry/{service_category_id}`
+    * e.g. https://api.servicedirect.com/cpl/industry/2
```
Comment (john-servicedirect) [inline README.md:40]: Is returning the highest CPL what we want? Should we return the highest Bid instead? Do we have people using this endpoint?

Comment (nimlevy) [reply README.md:40]: We do have people using the endpoint. We don't track it in a DB table, but you can see many records in Apache's access logs.

We could enhance to endpoint to accept an affiliate key, in which case it'll return the highest bid.

Comment (nimlevy) [reply README.md:40]: @john-servicedirect what are your thoughts on enhancing the route with bid specific values based on the ad network?
Or should we merge these changes now, and open a new task to tackle this possible change?

Comment (john-servicedirect) [reply README.md:40]: @nimlevy I think we merge and go with what we've got for now. 

Perhaps 1 copy change: "Find the highest possible Payout per Zip Code:"

Comment (nimlevy) [reply README.md:40]: done

**Pull request merged**